src/Hive.IO/Hive.IO/Building/Sia2024Record.cs

```diff
@@ -135,14 +201,26 @@ public string ToJson()
                 {"Kosten opake Bauteile", OpaqueCost},
                 {"Kosten transparente Bauteile", TransparentCost},
                 {"Emissionen opake Bauteile", OpaqueEmissions},
-                {"Emissionen transparente Bauteile", TransparentEmissions}
+                {"Emissionen transparente Bauteile", TransparentEmissions},
+
+                {"U-Wert Boeden", UValueFloors },
+                {"U-Wert Daecher", UValueRoofs },
+                {"U-Wert Walls", UValueWalls },
+                {"Kosten Boeden", CostFloors},
+                {"Kosten Daecher", CostRoofs },
+                {"Kosten Waende", CostWalls },
+                {"Emissionen Boeden", EmissionsFloors },
+                {"Emissionen Daecher", EmissionsRoofs },
+                {"Emissionen Waende", EmissionsWalls }
```

Comment:
I'm just thinking... with this approach we won't be able to assign individual surfaces. Example: we have 4 walls, 2 walls are made of concrete, 1 of wood and 1 of steel. How would that be stored in this json format?

Reply:
but then, on a second thought... will we really ever have that feature in hive?